.eslintrc.js

```diff
@@ -39,9 +40,6 @@ module.exports = {
     "overrides": [
         {
             "files": [ "*.vue" ],
-            "rules": {
-                "vue/html-indent": [2, 4],
-            }
```

Comment:
Those instructions include a command line tool that spits out conflicting rules in your config. This was flagged as one that conflicts with prettier. One can have a look at the commit I linked in the body of the PR to see an example change to mustache expressions in vue templates and see if you're groovy with it.

Reply:
I'm confused on why these rules conflict, does prettier not support 4-space indent?

Reply:
I'm confused on why these rules conflict, does prettier not support 4-space indent?

Reply:
Yes, to be clear, prettier will still take care of 4-space indent in the general case, with or without this rule on the eslint side. The conflict is more subtle. With code like this:

```vue
<template>
    <span>
        {{
            slotProps.node.data.name
            ?? bestLabel(slotProps.node.data, selectedLanguage.code).value    
        }}
    </span>
</template>
```

... leaving the eslint rule as is will insist on an additional indent:

```vue
<template>
    <span>
        {{
            slotProps.node.data.name
                ?? bestLabel(slotProps.node.data, selectedLanguage.code).value    
        }}
    </span>
</template>
```

Whereas prettier will insist on:
```vue
<template>
    <span>
        {{
            slotProps.node.data.name ??
            bestLabel(slotProps.node.data, selectedLanguage.code).value
        }}
    </span>
</template>
```

Then you end up with the headache where prettier ran on pre-commit, you push, and eslint fails.

Basically, if we're using a formatter at all, we need to let the formatter have the final say about indentation for better or worse.